templates/loadplugin.conf.erb

```diff
@@ -2,6 +2,9 @@
 <% if scope.lookupvar('collectd::collectd_version_real') and (scope.function_versioncmp([scope.lookupvar('collectd::collectd_version_real'), '4.9.4']) >= 0) -%>
 <LoadPlugin <%= @plugin %>>
   Globals <%= @globals %>
+<% if @flushinterval and scope.lookupvar('collectd::collectd_version_real') and (scope.function_versioncmp([scope.lookupvar('collectd::collectd_version_real'), '5.2']) >= 0) -%>
+  FlushInterval <%= @flushinterval %>
```

Comment:
please add a unit test to validate that the condition works

Reply:
done!